src/containers/community/CommunityPosts.js

```diff
@@ -42,10 +43,10 @@ const getFeedModule = (community, currentUser, moduleChoice) => {
 
   if (showPopularSkills && showPostPrompt) {
     module.component = moduleChoice
-    ? <PopularSkillsModule community={community} />
+    ? <PopularSkillsModule slug={community.slug} />
```

Comment:
i don't quite follow here -- why would this `community` not have `popularSkills`?